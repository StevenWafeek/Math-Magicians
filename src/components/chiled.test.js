import React from 'react';
import  { render } from '@testing-library/react';
import Display from './chiled';
import '@testing-library/jest-dom/extend-expect';
import { prettyDOM } from '@testing-library/dom';

describe('Calculator', () => {
 const calculator = render(<Display />)
test('calculator is rendering',() => {
    expect(prettyDOM(calculator.container)).toMatchSnapshot()
});

});