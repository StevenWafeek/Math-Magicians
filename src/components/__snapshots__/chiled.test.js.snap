// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Calculator calculator is rendering 1`] = `
"[36m<div>[39m
  [36m<div[39m
    [33mclass[39m=[32m\\"display\\"[39m
  [36m/>[39m
[36m</div>[39m"
`;
